scripts/log_reports.py

```diff
@@ -1,123 +1,140 @@
 import json, os
+import argparse
 from pathlib import Path
 from datetime import date
 from tabulate import tabulate
 
 
 MAX_LEN_MESSAGE = 2900  # slack endpoint has a limit of 3001 characters
 
-failed = []
-passed = []
-
-group_info = []
-
-total_num_failed = 0
-empty_file = False or len(list(Path().glob("*.log"))) == 0
-
-total_empty_files = []
-
-for log in Path().glob("*.log"):
-    section_num_failed = 0
-    i = 0
-    with open(log, "r") as f:
-        for line in f:
-            line = json.loads(line)
-            i += 1
-            if line.get("nodeid", "") != "":
-                test = line["nodeid"]
-                if line.get("duration", None) is not None:
-                    duration = f'{line["duration"]:.4f}'
-                    if line.get("outcome", "") == "failed":
-                        section_num_failed += 1
-                        failed.append([test, duration, log.name.split('_')[0]])
-                        total_num_failed += 1
-                    else:
-                        passed.append([test, duration, log.name.split('_')[0]])
-        empty_file = i == 0
-    group_info.append([str(log), section_num_failed, failed])
-    total_empty_files.append(empty_file)
-    os.remove(log)
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--slack_channel_name",
+    default=None
+)
+
+
+def main(slack_channel_name=None):
+    # Default it to the peft daily ci
+    if slack_channel_name is None:
+        slack_channel_name = "peft-ci-daily"
+
     failed = []
-no_error_payload = {
-    "type": "section",
-    "text": {
-        "type": "plain_text",
-        "text": "🌞 There were no failures!" if not any(total_empty_files) else "Something went wrong there is at least one empty file - please check GH action results.",
-        "emoji": True
-    }
-}
+    passed = []
```

Comment:
Since I'm lazy: Did you change any of the code in the `main` function, or is it just indented and the rest is the same?

Reply:
No no, I just idented it and the rest is all same